modules/ROOT/pages/tutorial-basic-provisioning.adoc

```diff
@@ -0,0 +1,268 @@
+
+Luckily `fcct` outputs valid Ignition configs. However, if you are tweaking the config after `fcct`, or manually creating Ignition configs, you will have to verify that the config format is valid with `ignition-validate`:
+
+[source,bash]
+----
+ignition-validate simple.ign && echo 'success!'
+----
+
+== Booting Fedora CoreOS
+
+Now that we have an Ignition config, we can boot a virtual machine with it. This tutorial is based on QEMU but you should be able to use the same Ignition config on all the platforms supported by Fedora CoreOS.
+
+We use `virt-install` to create a new Fedora CoreOS virutal machine with a specific config:
+
+[source,bash]
+----
+# Setup the correct SELinux label to allow access to the config
+chcon --verbose unconfined_u:object_r:svirt_home_t:s0 simple.ign
+
+# Start a temporary virtual machine
+virt-install --name=fcos --vcpus=2 --ram=2048 --import \
+    --network=bridge=virbr0 --graphics=none \
+    --qemu-commandline="-fw_cfg name=opt/com.coreos/config,file=${PWD}/simple.ign" \
+    --disk=size=20,backing_store=${PWD}/fedora-coreos.qcow2
+----
+
+The `virt-install` command will start an instance named `fcos` from the `fedora-coreos.qcow2` image using the `simple.ign` Ignition config. It will auto-attach the serial console of the machine so you'll be able to see the image bootup messages.
+
+We use the `backing_store` option to `virt-install --disk` to quickly create a new disk image and avoid writing to the original image we have downloaded. This new disk image can be easily thrown away.
+
+Once the machine is booted up you should see a few promps and then you should be automatically logged in and presented with a bash shell:
+
+[source,bash]
+----
+[  OK  ] Started rpm-ostree System Management Daemon.
+
+Fedora CoreOS 32.20200715.3.0
+Kernel 5.7.8-200.fc32.x86_64 on an x86_64 (ttyS0)
+
+SSH host key: SHA256:XlbayjbgDKNoAAHQxsEL5Q7BdwLxxWSw4NXN9SALLmo (ED25519)
+SSH host key: SHA256:3sx5jseteO4BvdOMWIi0J4koQL015mLonnD0UPTtnZk (ECDSA)
+SSH host key: SHA256:K0fn5/TMJOoMs7Fu7RRkE7IBEf2t8OYCfVaVc+GJWGs (RSA)
+ens2: 192.168.122.127 fe80::5054:ff:feb9:3d97
+Ignition: user provided config was applied
+No ssh authorized keys provided by Ignition or Afterburn
+tutorial login: core (automatic login)
+
+[core@tutorial ~]$
+----
+
+Let's verify that our configuration has been correctly applied. As we were automatically logged in to the terminal, we can safely assume that the systemd dropin has been created:
+
+[source,bash]
+----
+[core@tutorial ~]$ systemctl cat serial-getty@ttyS0.service
+# /usr/lib/systemd/system/serial-getty@.service
+...
+
+# /etc/systemd/system/serial-getty@ttyS0.service.d/autologin-core.conf
+[Service]
+# Override Execstart in main unit
+ExecStart=
+# Add new Execstart with `-` prefix to ignore failure`
+ExecStart=-/usr/sbin/agetty --autologin core --noclear %I $TERM
+----
+
+We can also check that the hostname has correctly been set:
+
+[source,bash]
+----
+[core@tutorial ~]$ hostname
+tutorial
+----
```

Comment:
Might be cool if we go a little more explicit here. So we first check the file got written

```
$ cat /etc/hostname
tutorial
```

Then run `hostnamectl` which is a little more fancy:

```
$ hostnamectl 
   Static hostname: n/a
Transient hostname: foo.example.com
         Icon name: computer-desktop
           Chassis: desktop
        Machine ID: acc83b1c3cd04189897696fe50fd1932
           Boot ID: 5e14930ff3504e3da0fb26f6325c0cb6
  Operating System: Fedora CoreOS 32.20200726.2.0
       CPE OS Name: cpe:/o:fedoraproject:fedora:32
            Kernel: Linux 5.7.10-201.fc32.x86_64
      Architecture: x86-64
```